Doc/library/tkinter.rst

```diff
@@ -11,9 +11,8 @@
 --------------
 
 The :mod:`tkinter` package ("Tk interface") is the standard Python interface to
-the Tk GUI toolkit.  Both Tk and :mod:`tkinter` are available on most Unix
-platforms, as well as on Windows systems.  (Tk itself is not part of Python; it
-is maintained at ActiveState.)
+the Tcl/Tk GUI toolkit.  Both Tk and :mod:`tkinter` are available on most Unix
+platforms, including macOS, as well as on Windows systems.
```

Comment:
```suggestion
platforms (including macOS) as well as on Windows systems.
```
IMO, this flows better with the following "as well as ..."

Reply:
Disagree, as you need the comma before "as well as" for the external flow anyway, as you can see on the red part of the diff.